Package.swift

```diff
@@ -227,10 +237,10 @@ platforms = nil
 #else
 platforms = [
     // we require the 'distributed actor' language and runtime feature:
-    .iOS(.v16),
-    .macOS(.v14),
-    .tvOS(.v16),
-    .watchOS(.v9),
+    .iOS(.v18),
+    .macOS(.v15),
+    .tvOS(.v18),
+    .watchOS(.v11),
```

Comment:
While this is not required, it will give some goodies like Synchronization

Reply:
that's ok